specification/dynatrace/resource-manager/Dynatrace.Observability/stable/2023-04-27/dynatrace.json

```diff
@@ -0,0 +1,2244 @@
+          "description": "Id of the Marketplace SaaS Resource"
+        },
+        "planId": {
+          "type": "string",
+          "description": "Id of the plan"
+        },
+        "marketplaceSubscriptionStatus": {
+          "$ref": "#/definitions/MarketplaceSubscriptionStatus",
+          "description": "Marketplace subscription status"
+        }
+      },
+      "description": "Marketplace SaaS resource details linked to the given tenant Id"
+    },
+    "VMExtensionPayload": {
+      "type": "object",
+      "properties": {
+        "ingestionKey": {
+          "type": "string",
+          "description": "Ingestion key of the environment"
+        },
+        "environmentId": {
+          "type": "string",
+          "description": "Id of the environment created"
+        }
+      },
+      "description": "Response of payload to be passed while installing VM agent."
+    },
+    "VMHostsListResponse": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/VMInfo"
+          },
+          "description": "The items on this page",
+          "x-ms-identifiers": []
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The link to the next page of items"
+        }
+      },
+      "description": "Response of a list VM Host Operation."
+    },
+    "AppServiceInfo": {
+      "type": "object",
+      "properties": {
+        "resourceId": {
+          "type": "string",
+          "description": "App service resource ID"
+        },
+        "version": {
+          "type": "string",
+          "description": "Version of the Dynatrace agent installed on the App Service."
+        },
+        "monitoringType": {
+          "$ref": "#/definitions/MonitoringType",
+          "description": "The monitoring mode of OneAgent"
+        },
+        "autoUpdateSetting": {
+          "$ref": "#/definitions/AutoUpdateSetting",
+          "description": "Update settings of OneAgent."
+        },
+        "updateStatus": {
+          "$ref": "#/definitions/UpdateStatus",
+          "description": "The current update status of OneAgent."
+        },
+        "availabilityState": {
+          "$ref": "#/definitions/AvailabilityState",
+          "description": "The availability state of OneAgent."
+        },
+        "logModule": {
+          "$ref": "#/definitions/LogModule",
+          "description": "Tells whether log modules are enabled or not"
+        },
+        "hostGroup": {
+          "type": "string",
+          "description": "The name of the host group"
+        },
+        "hostName": {
+          "type": "string",
+          "description": "The name of the host"
+        }
+      },
+      "description": "Details of App Services having Dynatrace OneAgent installed"
+    },
+    "MetricsStatusResponse": {
+      "type": "object",
+      "properties": {
+        "azureResourceIds": {
```

Comment:
What does Status here mean? I see properties here include array of resource IDs but no status. Can you help me understand the scenario?

Reply:
The array is a list of azure resource ids that are sending metrics

Reply:
Then is metrics status is proper naming here? It might confuse Cx. May be metrics emitters or something similar?

Reply:
@samsaha-ms For us, this array tells the status of resource which are emitting metrics. Can we keep it like that for our understanding?

Reply:
You should design APIs from Cx point of view too as they are the consumer for this APIs.

Reply:
This API we will be using internally and will ask not to publish to publicly.